sdk/src/main/java/com/qonversion/android/sdk/automations/QAutomationsManager.kt

```diff
@@ -163,11 +181,15 @@ class QAutomationsManager @Inject constructor(
     }
 
     private fun sendPushToken(token: String) {
-        repository.setPushToken(token)
+        repository.sendPushToken(token)
 
         pendingToken = null
     }
```

Comment:
Here is one mistake of two sources of truth - we store the token to preferences when we can't send it, but never clear when we've successfully sent it. Thus we will send the token to API after every launch.

Reply:
No, because we clear the pending push token after a successful request